packages/pages/src/dev/server/ssr/pageLoader.ts

```diff
@@ -117,5 +108,5 @@ export const pageLoader = async ({
     relativePrefixToRoot: getRelativePrefixToRootFromPath(path),
   };
 
-  return { template, Component, props: templateRenderProps };
+  return { template, Component, render, props: templateRenderProps };
```

Comment:
would it be better to just return the `TemplateModuleInternal` rather than pieces of it?

Reply:
It's already defined in the caller so I can just use it from there and not return as many things.